processing/data_collection/gazette/spiders/sc_florianopolis.py

```diff
@@ -0,0 +1,73 @@
+import re
+from datetime import date, datetime
+
+from dateparser import parse
+from dateutil.relativedelta import relativedelta
+from scrapy import FormRequest, Spider
+
+from gazette.items import Gazette
+
+
+class ScFlorianopolisSpider(Spider):
+    name = 'sc_florianopolis'
+    URL = 'http://www.pmf.sc.gov.br/governo/index.php?pagina=govdiariooficial'
+    MUNICIPALITY_ID = '4205407'
+    AVAILABLE_FROM = date(2009, 6, 1)
+
+    def start_requests(self):
+        """The City Hall website publish the gazettes in a page with a form
+        that allow users to browse through different years and months. This
+        form sends requests via POST, so this method emulates a series of these
+        POSTs.
+        @url http://www.pmf.sc.gov.br/governo/index.php?pagina=govdiariooficial
+        @returns requests 1
+        """
+        target = date.today()
+        while target >= self.AVAILABLE_FROM:
```

Comment:
Cool that you gave preference to start collecting most recent gazettes before going to previous years.